src/veh_interact.cpp

```diff
@@ -1520,6 +1587,113 @@ void veh_interact::display_list(size_t pos, std::vector<vpart_info> list)
     wrefresh (w_list);
 }
 
+/**
+ * Used when installing parts.
+ * Opens up w_details containing info for part currently selected in w_list.
+ */
+void veh_interact::display_details(const vpart_info &part)
+{
+    if( &part == NULL ) return;
+
+    if (w_details == NULL) { // create details window first if required
+
+        if (vertical_menu) { // clear rightmost blocks of w_stats in vertical/hybrid mode to avoid overlap
+            for (size_t i = 0; i < w_stats->height; i++) {
+                mvwhline(w_stats, i, 34, ' ', w_stats->width - 34);
+            }
+            wrefresh(w_stats);
+        }
+
+        // covers right part of w_name and w_stats in vertical/hybrid, lower block of w_stats in horizontal mode
+        const int details_y = vertical_menu ? w_name->y : w_stats->y + 7;
+        const int details_x = vertical_menu ? w_list->x : w_stats->x;
+
+        const int details_h = vertical_menu ? 6 : 7; // 6 lines in vertical/hybrid, 7 lines in horizontal mode
+        const int details_w = w_grid->x + w_grid->width - details_x;
+
+        w_details = newwin(details_h, details_w, details_y, details_x);
+    }
+    else {
+        werase(w_details);
+   }
+
+    wborder(w_details, LINE_XOXO, LINE_XOXO, LINE_OXOX, LINE_OXOX, LINE_OXXO, LINE_OOXX, LINE_XXOO, LINE_XOOX);
+
+    int column_width = w_details->width / 2; // displays data in two columns
```

Comment:
Haha ok, I was wondering why all the window properties were also members of ```veh_interact```. I'll set the project up on Linux to fix and test the build first thing tomorrow.